website/blog/2021-11-18-2.5.0.md

```diff
@@ -0,0 +1,366 @@
+
+<!-- prettier-ignore -->
+```scss
+// Input
+@use 'library' with (
+  $black: #222,
+  $border-radius: 0.1rem $font-family: 'Helvetica, sans-serif'
+);
+
+// Prettier 2.4
+@use 'library' with
+  ($black: #222, $border-radius: 0.1rem $font-family: 'Helvetica, sans-serif');
+
+// Prettier 2.5
+@use 'library' with (
+  $black: #222,
+  $border-radius: 0.1rem $font-family: 'Helvetica, sans-serif'
+);
+
+```
+
+#### Fix `@forward with` formatting error ([#11683](https://github.com/prettier/prettier/pull/11683) by [@sriramarul](https://github.com/sriramarul), [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+<!-- prettier-ignore -->
+```scss
+// Input
+@forward 'foo.scss' with ($components: red);
+
+// Prettier 2.4
+TypeError: Cannot read properties of undefined (reading 'type')
+
+// Prettier 2.5
+@forward "foo.scss" with (
+  $components: red
+);
+
+```
+
+### HTML
+
+#### Reverts [#7865](https://github.com/prettier/prettier/pull/7865), keeping class names on the same line ([#11827](https://github.com/prettier/prettier/pull/11827) by [@jlongster](https://github.com/jlongster))
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div
+  class="SomeComponent__heading-row d-flex flex-column flex-lg-row justify-content-start justify-content-lg-between align-items-start align-items-lg-center"
+></div>
+
+<!-- Prettier 2.4 -->
+<div
+  class="
+    SomeComponent__heading-row
+    d-flex
+    flex-column flex-lg-row
+    justify-content-start justify-content-lg-between
+    align-items-start align-items-lg-center
+  "
+></div>
+
+<!-- Prettier 2.5 -->
+<div
+  class="SomeComponent__heading-row d-flex flex-column flex-lg-row justify-content-start justify-content-lg-between align-items-start align-items-lg-center"
+></div>
+
+```
+
+### Ember / Handlebars
+
+#### Uses the opposite quote type for quotes inside mustache statements in attributes ([#11524](https://github.com/prettier/prettier/pull/11524) by [@bmaehr](https://github.com/bmaehr))
+
+<!-- prettier-ignore -->
+```hbs
+{{!-- Input --}}
+<div title="{{t 'login.username.description'}}" />
+
+{{!-- Prettier 2.5 --}}
+<div title="{{t 'login.username.description'}}" />
+
+{{!-- Prettier 2.4 --}}
+<div title="{{t "login.username.description"}}" />
+
+```
+
+### Markdown
+
+#### Keep trailing commas for type parameters in embedded TSX ([#11685](https://github.com/prettier/prettier/pull/11685) by [@sosukesuzuki](https://github.com/sosukesuzuki))
+
+<!-- Optional description if it makes sense. -->
+
+<!-- prettier-ignore -->
+````md
+<!-- Input  -->
+```tsx
+const test = <T,>(value: T) => {};
+```
+
+<!-- Prettier 2.4 -->
+```tsx
+const test = <T>(value: T) => {};
+```
+
+<!-- Prettier 2.5 -->
+```tsx
+const test = <T,>(value: T) => {};
+```
+````
+
+### CLI
+
+#### Add new CLI option `debug-print-ast` ([#11514](https://github.com/prettier/prettier/pull/11514) by [@sosukesuzuki](https://github.com/sosukesuzuki))
```

Comment:
According to [CHANGELOG](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#241), this feature has already been released in 2.4.1 🤔